### DISCUSSION TITLE
multiple images watermark ??

### DISCUSSION THREAD

**Discussion by hasbycs** [razztyfication/vue-drawing-canvas#12] [Ideas]
can I add multiple images watermark ??

**Comment by razztyfication** [comment]
No you can't, how can you came up with this idea?

**Reply by hasbycs** [reply]
I need to insert multiple icon/image including customization like size or position

**Reply by razztyfication** [reply]
so let's say watermark props is Array of Object. did watermark Object fulfill your criteria?

**Reply by hasbycs** [reply]
yeah, like thats

**Reply by razztyfication** [reply]
let me think about it, I didn't want to change watermark object.

what if I add `additionalImages` and/or `additionalText` as props? 
it will accept array of watermark object so you can achieve your objectives

**Comment by razztyfication** [comment]
Sorry for keep you waiting

can you try version `1.0.9-rc.2`?
I add `additionalImages` as props and accept array of watermark object

please be aware of performance issues and let me know your opinion
